modules/mobile_session.lua

```diff
@@ -101,8 +101,15 @@ function mt.__index:ExpectNotification(funcName, ...)
     data.sessionId == self.sessionId
   end
   args = table.pack(...)
-  if #args[1] > 0 or args[1].n == 0 then
-    args = args[1]
+
+  if #args ~= 0 then
```

Comment:
@LuxoftAKutsan changed in https://github.com/AMikhniev/sdl_atf/commit/4050d9cabc78b7ba71f0aae271d904159b5ef707